### ISSUE TITLE
npx roger "watch" will watch file changes

### ISSUE THREAD

**Issue by amirebrahimi** [penrose/penrose#577]
**Is your feature request related to a problem? Please describe.**
I'd expect that a "watch" command would watch files that changed, but it doesn't seem the case with `roger`.

**Describe the solution you'd like**
Have `roger` reload files that have changed, so I don't have to stop the command and restart.

**Describe alternatives you've considered**
Not sure of any way to work around this at the moment.

**Additional context**
It's pretty simple to repro. Start up a session, then make a change to a `.sub` file and Resample in the web UI. You'll notice that changes are not reflected.

**Comment by maxkrieger** [comment]
What OS/setup are you using? This sounds like a bug - the expected behavior is what you describe! We use the [chokidar](https://github.com/paulmillr/chokidar) library.

**Comment by amirebrahimi** [comment]
I'm using Ubuntu 20.04 running on top of WSL2/Windows10, but for all effective purposes just think of it as Linux. Unfortunately, I don't see those messages showing that the file was updated. I only see the "starting on port 9160".

**Comment by amirebrahimi** [comment]
I decided to try to reproduce this on native Windows w/ a Conda environment through MINGW32. I wasn't able to. I no longer have the high CPU utilization that I was seeing on WSL2/Ubuntu and the file watch seems to be working now. 

So, then I went back to WSL2/Ubuntu and decided to re-clone the penrose repository to a native Ubuntu filesystem and then copy my files locally to the native filesystem. Lo and behold the file watcher works.

My conclusion is that WSL2 causes issues with the file watcher when using the mixed filesystem support in WSL2 (i.e. accessing a Windows partition from Ubuntu).

I'll go ahead and close this issue.
